client/src/services/scripts/selectors.tsx

```diff
@@ -0,0 +1,5 @@
+import { IReduxStore } from 'types';
+
+const getScriptList = (state: IReduxStore) => state.scriptList;
```

Comment:
What do you think about destructuring (`{ data: scriptsList }`) here?

Reply:
will do